string room_name
---
geometry_msgs/Point point
